Documentation/install/upgrade.rst

```diff
@@ -209,35 +290,90 @@ Both files are dedicated to "\ |SCM_BRANCH|" for each Kubernetes version.
       $ kubectl apply -f \ |SCM_WEB|\/examples/kubernetes/1.11/cilium-rbac.yaml
       $ kubectl apply -f \ |SCM_WEB|\/examples/kubernetes/1.11/cilium-ds.yaml
 
+.. _upgrade_version:
 
-You can also substitute the desired Cilium version number for vX.Y.Z in the
-command below, but be aware that copy of the spec file stored in Kubernetes
-might run out-of-sync with the CLI flags, or options, specified by each Cilium
-version.
+Direct version upgrade
+""""""""""""""""""""""
 
-::
+You can alternatively substitute the version ``vX.Y.Z`` for the desired Cilium
+version number in the command below, but be aware that copy of the spec file
+stored in Kubernetes might run out-of-sync with the CLI flags, or options,
+specified by each Cilium version.
+
+.. code-block:: shell-session
+
+    $ kubectl set image daemonset/cilium -n kube-system cilium-agent=docker.io/cilium/cilium:vX.Y.Z
 
-    kubectl set image daemonset/cilium -n kube-system cilium-agent=docker.io/cilium/cilium:vX.Y.Z
+Monitor the upgrade procedure
+"""""""""""""""""""""""""""""
 
 To monitor the rollout and confirm it is complete, run:
 
-::
+.. code-block:: shell-session
 
-    kubectl rollout status daemonset/cilium -n kube-system
+    $ kubectl rollout status daemonset/cilium -n kube-system
 
 To undo the rollout via rollback, run:
```

Comment:
I think this should be in a different section, distracted users my run this blindly and they will undo the upgrade process in a single line without noticing.

Reply:
I could put this in a nested "note" box or something so it's highlighted that this should only be necessary in unusual circumstances?

Reply:
Actually, your original suggestion to put it in its own section would be better IMO.

Reply:
@joestringer There's already a Downgrade section below

Reply:
Downgrade may occur during upgrade, or it could occur some time later (eg, someone realises a problem after 1 day). These instructions are here in case you need them during upgrade, whereas I've thought of the `Downgrade` section as somewhere that handles the more general case. That said, I think the downgrade section may need some additional love.